src/main/java/HomePageAmazon.java

```diff
@@ -0,0 +1,24 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class HomePageAmazon {
+
+    WebDriver driver;
+
+    public HomePageAmazon(WebDriver driver){
+        this.driver = driver;
+    }
+
+    public HomePageAmazon writeNintendo() {
+        By Search = By.id("twotabsearchtextbox");
+        driver.findElement(Search).sendKeys("Nintendo Switch");
```

Comment:
The text "Nintendo Switch" must not be here, this test must be in the test, not in the page object